services/slam/client_test.go

```diff
@@ -115,15 +115,15 @@ func TestClientWorkingService(t *testing.T) {
 		test.That(t, spatial.PoseAlmostEqual(poseSucc, pose), test.ShouldBeTrue)
 		test.That(t, componentRef, test.ShouldEqual, componentRefSucc)
 
-		// test get point cloud map stream
```

Comment:
Optional removal, I'm in favor to keep pattern with server tests

Reply:
this is fine, we have a [ticket](https://viam.atlassian.net/browse/RSDK-2130?atlOrigin=eyJpIjoiOTg4M2NhYjA2OGU0NDFkOGEzNTQ4YTFhZGJiNDBmOWYiLCJwIjoiaiJ9) to make server tests into individual t.runs, which should probably include these tests too

Reply:
I agree with john. Lets address client tests not being in t.Runs in the ticket he linked.

Reply:
[note to self] we took out the word "stream" everywhere that we aren't specifically testing the streaming aspect of the api